main/postfix/APKBUILD

```diff
@@ -2,33 +2,49 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=postfix
-pkgver=3.3.1
-pkgrel=4
+pkgver=3.3.2
+pkgrel=0
 pkgdesc="Secure and fast drop-in replacement for Sendmail (MTA)"
 url="http://www.postfix.org/"
 arch="all"
 license="IPL-1.0"
+#license="EPL-2.0"
```

Comment:
that looks like forgotten

Reply:
Do you have a suggestion for software with multiple licenses to choose from?

Reply:
All of them should be listed, check php7 as example

Reply:
I checked php7 and that is not a case of choice of license. That situation is all of those licenses apply because of the modules included.

I'm not sure how best to handle this.
> Dual license: in addition to the historical IBM Public License 1.0, Postfix is now also distributed with the more recent Eclipse Public License 2.0. Recipients can choose to take the software under the license of their choice. Those who are more comfortable with the IPL can continue with that license.

http://www.postfix.org/announcements/postfix-3.3.0.html

Reply:
I found `zstd` has the same situation and lists both. I'll do the same.

@ncopa This really should be handled some way that makes it clear the license is a choice. Perhaps `alpine-devel` is a better place to discuss the situation.